sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormTrainingAsyncClientJavaDocCodeSnippets.java

```diff
@@ -0,0 +1,168 @@
+
+        });
+        // END: com.azure.ai.formrecognizer.FormTrainingAsyncClient.getCustomModel#string
+    }
+
+    /**
+     * Code snippet for {@link FormTrainingAsyncClient#getCustomModelWithResponse}
+     */
+    public void getCustomModelWithResponse() {
+        // BEGIN: com.azure.ai.formrecognizer.FormTrainingAsyncClient.getCustomModelWithResponse#string
+        String modelId = "{model_id}";
+        formTrainingAsyncClient.getCustomModelWithResponse(modelId).subscribe(response -> {
+            CustomFormModel customFormModel = response.getValue();
+            System.out.printf("Model Id: %s%n", customFormModel.getModelId());
+            System.out.printf("Model Status: %s%n", customFormModel.getModelStatus());
+            customFormModel.getSubModels().forEach(customFormSubModel ->
+                customFormSubModel.getFieldMap().forEach((key, customFormModelField) ->
+                    System.out.printf("Model Type Id: %s Field Text: %s Field Accuracy: %s%n",
+                        key, customFormModelField.getName(), customFormModelField.getAccuracy())));
+        });
+        // END: com.azure.ai.formrecognizer.FormTrainingAsyncClient.getCustomModelWithResponse#string
+    }
+
+    /**
+     * Code snippet for {@link FormTrainingAsyncClient#getAccountProperties}
+     */
+    public void getAccountProperties() {
+        // BEGIN: com.azure.ai.formrecognizer.FormTrainingAsyncClient.getAccountProperties
+        formTrainingAsyncClient.getAccountProperties().subscribe(accountProperties -> {
+            System.out.printf("Account properties limit: %s%n", accountProperties.getLimit());
+            System.out.printf("Account properties count: %d%n", accountProperties.getCount());
+        });
+        // END: com.azure.ai.formrecognizer.FormTrainingAsyncClient.getAccountProperties
+    }
+
+    /**
+     * Code snippet for {@link FormTrainingAsyncClient#getAccountPropertiesWithResponse}
+     */
+    public void getAccountPropertiesWithResponse() {
+        // BEGIN: com.azure.ai.formrecognizer.FormTrainingAsyncClient.getAccountPropertiesWithResponse
+        formTrainingAsyncClient.getAccountPropertiesWithResponse().subscribe(response -> {
+            AccountProperties accountProperties = response.getValue();
+            System.out.printf("Account properties limit: %s%n", accountProperties.getLimit());
+            System.out.printf("Account properties count: %d%n", accountProperties.getCount());
+        });
+        // END: com.azure.ai.formrecognizer.FormTrainingAsyncClient.getAccountPropertiesWithResponse
+    }
+
+    /**
+     * Code snippet for {@link FormTrainingAsyncClient#deleteModel}
+     */
+    public void deleteModel() {
+        // BEGIN: com.azure.ai.formrecognizer.FormTrainingAsyncClient.deleteModel#string
+        String modelId = "{model_id}";
+        formTrainingAsyncClient.deleteModel(modelId).subscribe(val ->
+            System.out.printf("Model ID = %s is deleted%n", modelId));
```

Comment:
No need to subscribe on void.
just update this to formTrainingAsyncClient.deleteModel("{modelId}")

Reply:
FYI, when app config do deletion, like deleting a configuration, we do return object to user.

does method call trigger if without subscribe?